exasol/toolbox/tools/workflow.py

```diff
@@ -0,0 +1,187 @@
+import difflib
+import io
+from contextlib import ExitStack
+from importlib import resources
+from pathlib import Path
+from typing import (
+    Mapping,
+    Union,
+    Any,
+)
+
+import typer
+from rich.columns import Columns
+from rich.console import Console
+from rich.syntax import Syntax
+
+stdout = Console()
+stderr = Console(stderr=True)
+
+CLI = typer.Typer()
+
+
+def _workflows() -> Mapping[str, Any]:
+    pkg = "exasol.toolbox.templates.github.workflows"
+
+    def _normalize(name: str) -> str:
+        name, ext = name.split(".")
+        return name
+
+    return {_normalize(w.name): w for w in resources.files(pkg).iterdir()}  # type: ignore
```

Comment:
importlib.resources.files is only in the stdlib since python 3.9 before you need to install https://pypi.org/project/importlib-resources/
https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files

iterdir is also only in the stdlib since python 3.9 https://docs.python.org/3/library/importlib.resources.abc.html